src/nvim/charset.h

```diff
@@ -34,4 +34,15 @@ typedef enum {
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "charset.h.generated.h"
 #endif
+
+static inline bool vim_isbreak(int c)
+  REAL_FATTR_CONST
+  REAL_FATTR_ALWAYS_INLINE;
+
+/// Check if `c` is one of the characters in 'breakat'.
+/// Used very often if 'linebreak' is set
+static inline bool vim_isbreak(int c)
+{
+  return breakat_flags[(char_u)c];
```

Comment:
it's failing the "single include" check:

```
In file included from src/nvim/auto/charset.h.test-include.c:1:0:
/home/travis/build/neovim/neovim/src/nvim/charset.h: In function â€˜vim_isbreakâ€™:
/home/travis/build/neovim/neovim/src/nvim/charset.h:46:10: error: â€˜breakat_flagsâ€™ undeclared (first use in this function)
   return breakat_flags[(char_u)c];
          ^
/home/travis/build/neovim/neovim/src/nvim/charset.h:46:10: note: each undeclared identifier is reported only once for each function it appears in
```

I think `breakat_flags` could be moved to `option_defs.h`, then `#include "nvim/option_defs.h` from charset.h